src/components/templates/Auditorium/components/Section/Section.tsx

```diff
@@ -55,17 +64,18 @@ export const Section: React.FC<SectionProps> = ({ venue }) => {
     };
   }, [leaveSeat]);
 
-  const iframeInlineStyles: React.CSSProperties = useMemo(
+  const centralScreenInlineStyles: React.CSSProperties = useMemo(
     () => ({
-      width: `calc(${videoWidthInSeats} * (${SECTION_SEAT_SIZE} + ${SECTION_SEAT_SPACING}))`,
-      height: `calc(${videoHeightInSeats} * (${SECTION_SEAT_SIZE} + ${SECTION_SEAT_SPACING}))`,
-      minWidth: `calc(${videoWidthInSeats} * (${SECTION_SEAT_SIZE_MIN} + ${SECTION_SEAT_SPACING}))`,
-      minHeight: `calc(${videoHeightInSeats} * (${SECTION_SEAT_SIZE_MIN} + ${SECTION_SEAT_SPACING}))`,
+      width: `calc(${screenWidthInSeats} * (${SECTION_SEAT_SIZE} + ${SECTION_SEAT_SPACING}))`,
+      height: `calc(${screenHeightInSeats} * (${SECTION_SEAT_SIZE} + ${SECTION_SEAT_SPACING}))`,
+      minWidth: `calc(${screenWidthInSeats} * (${SECTION_SEAT_SIZE_MIN} + ${SECTION_SEAT_SPACING}))`,
+      minHeight: `calc(${screenHeightInSeats} * (${SECTION_SEAT_SIZE_MIN} + ${SECTION_SEAT_SPACING}))`,
```

Comment:
Will it not be simpler to move all the `calc()` properties inside `Section.scss` and just set the `--screen-width-in-seats`-like variables here?
Like:
```
  "--screen-width-in-seats": screenWidthInSeats,
  "--screen-height-in-seats":screenHeightInSeats,
```

Reply:
Thanks for the suggestion. Fixed in https://github.com/sparkletown/sparkle/pull/1339/commits/d2a708581428aa087efe8271fe99a7fa1029bdbe

Looks much cleaner